sopel/modules/meetbot.py

```diff
@@ -193,7 +193,7 @@ def is_chair(nick, channel):
 def startmeeting(bot, trigger):
     """
     Start a meeting.\
-    See [meetbot module usage]({% link _usage/meetbot-module.md %})
+    See [meetbot plugin usage]({% link _usage/meetbot-module.md %})
```

Comment:
This was already renamed, actually. I'll add suggestions to all of the occurrences.

I'm surprised that this didn't cause any website build errors, actually. Seems Jekyll is smarter than I thought.

```suggestion
    See [meetbot plugin usage]({% link _usage/meetbot-plugin.md %})
```